example.conf

```diff
@@ -10,3 +10,4 @@ user=
 key=
 auth_url=https://identity.api.rackspacecloud.com/v2.0/tokens
 limit=0
+replace_time=False
```

Comment:
Not sure if I like the name `replace_time` but if everyone is ok with it I am too.

I would mention it's the collection time that we're replacing with now() somehow, maybe `replace_metric_time` or even `replace_metric_time_on_flush`